{{cookiecutter.project_slug}}/Makefile

```diff
@@ -130,11 +133,14 @@ setup-release-tools-common:
 	npm install -g @semantic-release/exec@"^5.0.0"
 	npm install -g @semantic-release/git@"^9.0.0"
 
+{% if cookiecutter.hosting == "GitLab" -%}
 setup-release-tools-gitlab: setup-release-tools-common
 	npm install -g @semantic-release/gitlab@"^6.0.3"
 
+{%- else %}
 setup-release-tools-github: setup-release-tools-common
 	npm install -g @semantic-release/github@"^7.0.5"
+{%- endif %}
```

Comment:
`gitlab` and `github` qualifiers are not necessayr in make target name

Reply:
removed.